tests/fixtures/permissions/repository_permissions.yaml

```diff
@@ -12,5 +12,9 @@ opensafely/dummy_isaric:
   allow: ['isaric']
 opensafely/dummy_open_prompt:
   allow: ['open_prompt']
+opensafely/dummy_wl_clockstops:
+  allow: ['wl_clockstops']
+opensafely/dummy_wl_openpathways:
+  allow: ['wl_openpathways']
 opensafely/dummy_all:
-  allow: ['icnarc', 'ons', 'isaric', 'ons_cis', 'ukrr', 'open_prompt']
+  allow: ['icnarc', 'ons', 'isaric', 'ons_cis', 'ukrr', 'open_prompt', 'wl_clockstops', 'wl_openpathways']
```

Comment:
Should we take ons/ons_cis out of the tests too?

Reply:
We should! Thank you for spotting this.